docs/troubleshooting/02_rpcerror.md

```diff
@@ -0,0 +1,72 @@
+When a call to the chain_api is performed and fails, it will result in an RPCError object being generated to inform why the transaction failed.
+
+The RPCError object will error with the concise message, for instance 'Invalid transaction', but additional details can be found in both the `details` field and the `json` field.  The `json` field holds the entire json response from `nodeos` while the `details` field specifically holds the error object in the `json`.  This is typically in different places of the `json` depending on what endpoint is used in `nodeos` so the `details` field is available to quickly see the error information without needing to search through the `json` object.
```

Comment:
maybe

The RPCError object will error with the concise message, for instance 'Invalid transaction', but additional details can be found in both the `details` field and the `json` field. The `json` field holds the entire json response from `nodeos` while the `details` field specifically holds the error object in the `json`. This is typically in different places of the `json` depending on what endpoint is used in `nodeos` so the `details` field is available to quickly see the error information without needing to search through the `json` object.

to =>

The RPCError object will contain a concise error message, for instance 'Invalid transaction'. However additional details can be found in the `details` field and the `json` field. The `json` field holds the complete json response from `nodeos`. The `details` field specifically holds the error object in the `json` field. The data content of the `json` varies depending on the endpoint is used in `nodeos`. The `details` field is available to quickly see error information without needing to search through the `json` object.